File: LABORATORIOS/LABORATORIO3/Ejercicio1/Python/SumaLoteNumeros.py
def sumar_lote_numeros():
    n = int(input("Ingrese la cantidad de números en el lote: "))
    lote = []

    for i in range(n):
        numero = int(input("Ingrese un número: "))
        lote.append(numero)

    suma = sum(lote)
    return suma

resultado = sumar_lote_numeros()
print("La suma de los números es:", resultado)

File: LABORATORIOS/LABORATORIO3/Ejercicio1/Java/SumaLoteNumeros.java
import java.util.Scanner;

public class SumaLoteNumeros {
    public static int sumarLoteNumeros() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese la cantidad de números en el lote: ");
        int n = scanner.nextInt();
        int[] lote = new int[n];

        for (int i = 0; i < n; i++) {
            System.out.print("Ingrese un número: ");
            int numero = scanner.nextInt();
            lote[i] = numero;
        }

        int suma = 0;
        for (int i = 0; i < n; i++) {
            suma += lote[i];
        }

        return suma;
    }

    public static void main(String[] args) {
        int resultado = sumarLoteNumeros();
        System.out.println("La suma de los números es: " + resultado);
    }
}
